public/moreLinks.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <title>RERUM Geolocator</title>
+    <meta charset="UTF-8">
+    <link rel="icon" href="/images/favicon.ico" type="image/ico">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://unpkg.com/chota@latest">
+    <link rel="stylesheet" href="stylesheets/style.css">
+</head>
+
+<body is="geo-page">
+
+    <div class="container">
+        <div class="row" style="margin: 30px;">
+            <div class="col">
+                <h3> Additional Resources</h3>
+                <p class="intro">
+                    Below are additional resources related to Rerum Geolocator. Explore these links to learn more!
+                </p>
+            </div>
+        </div>
+        <ul>
+            <li>RERUM</li>
```

Comment:
We probably want these to be real links.  However, we didn't really flesh this out for this PR and I think this is good for this PR and maybe we can come back around to it when we know more about what we want to put here.